src/Explorer/Panes/PanelInfoErrorComponent.tsx

```diff
@@ -8,38 +8,47 @@ export interface PanelInfoErrorProps {
   link?: string;
   linkText?: string;
   openNotificationConsole?: () => void;
+  formError?: boolean;
 }
 
-export const PanelInfoErrorComponent: React.FunctionComponent<PanelInfoErrorProps> = (
-  props: PanelInfoErrorProps
-): JSX.Element => {
+export const PanelInfoErrorComponent: React.FunctionComponent<PanelInfoErrorProps> = ({
+  message,
+  messageType,
+  showErrorDetails,
+  link,
+  linkText,
+  openNotificationConsole,
+  formError = true,
+}: PanelInfoErrorProps): JSX.Element => {
   let icon: JSX.Element;
-  if (props.messageType === "error") {
-    icon = <Icon iconName="StatusErrorFull" className="panelErrorIcon" />;
-  } else if (props.messageType === "warning") {
-    icon = <Icon iconName="WarningSolid" className="panelWarningIcon" />;
-  } else if (props.messageType === "info") {
-    icon = <Icon iconName="InfoSolid" className="panelLargeInfoIcon" />;
+  if (messageType === "error") {
+    icon = <Icon iconName="StatusErrorFull" className="panelErrorIcon" data-testid="errorIcon" />;
```

Comment:
Use `aria-label` as selector for testing purposes if possible.

Reply:
done